Gui/GuiAppInstance.h

```diff
@@ -35,6 +35,10 @@
 #include <boost/scoped_ptr.hpp>
 #endif
 
+#ifdef SBK_RUN
+using namespace NATRON_ENUM;
```

Comment:
"using" at the top-level in an include is not a good practice, as it affects all subsequent includes too.

Can we prefix every enum in this file with NATRON_ENUM:: instead, as I did in the Engine includes?

Reply:
In fact this statement seems useless. Or am I missing something?